code/controllers/subsystem/timer.dm

```diff
@@ -503,11 +503,17 @@ SUBSYSTEM_DEF(timer)
  * If the timed event is tracking client time, it will be added to a special bucket.
  */
 /datum/timedevent/proc/bucketJoin()
-	// Generate debug-friendly name for timer
+	// Generate debug-friendly name for timer, more or less complex/expensive depending on defines
+#if defined(TIMER_DEBUG)
```

Comment:
```suggestion
#ifdef TIMER_DEBUG
```

this is an ignorable nit, but to me it feels like the whole `#ifdef`/`#endif` dichotomy is really good for pointing out compile-time macros? it probably doesn't matter since it does the same thing

Reply:
ahm okay